### PR TITLE
[Mobile app] new release

### LINKED ISSUE #10: Rating the app:  address needs to be updated, privacy policy should be pointed to the right page
Screenshot_20191219-172218

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Checklist
- [x] Update the android build number and version
- [x] Update the iOS version and build number.
- [x] Check your `.env.prod.`  Make sure you have set the right AirQo API (Ask the Security engineers for the key) and search Key [Link to credentials store](https://console.cloud.google.com/apis/credentials?project=airqo-250220) 
- [x] Make sure you have set the right Google Maps Key for android and iOS
  - [x]   Android => `prod-key.properties`
  - [x]   iOS => `mobile/ios/Runner/AppDelegate.swift`.
- [x] Check if you have the firebase file `GoogleService-Info.plist` set for iOS `mobile/ios/config`.
- [x] Check if you have the firebase file `google-services.json` set for android `mobile/ios/config`.
- [x] Check if you have `firebase_options.dart` file. 
- [x] Edit the release notes `mobile/android/release_notes.txt`

#### What are the relevant tickets?
- [MOB-156](https://airqoteam.atlassian.net/browse/MOB-156)
- Resolves #10 

[MOB-156]: https://airqoteam.atlassian.net/browse/MOB-156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mobile/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -819,7 +819,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 2.0.12;
+				MARKETING_VERSION = 2.0.14;
```
Comment (Baalmart) [inline mobile/ios/Runner.xcodeproj/project.pbxproj:822]: I am yet to appreciate all these version number changes we keep on making....perhaps this is the year! :)

**Review by Baalmart — APPROVED** [review]
LGTM

**Comment by Baalmart** [comment]
For east visibility, please remember to add a `ready to review` label the next time. @NoahNsimbe

**Comment by NoahNsimbe** [comment]
> For east visibility, please remember to add a `ready to review` label the next time. @NoahNsimbe

Oh..Sorry, it was accidentally removed. @Baalmart

**Pull request merged**